com.chartboost.helium/Editor/HeliumIntegrationChecker.cs

```diff
@@ -0,0 +1,329 @@
+        private const string HeliumPackageName = "com.chartboost.helium";
+        private const string HeliumSamplesInAssets = "Assets/Samples/Helium SDK";
+        private const string UnityAdsPackageName = "com.unity.ads";
+        private const string UnityAdsUncommentWindow = "unity-ads-uncomment-window";
+        private static readonly string HeliumSamplesMetaInAssets = $"{HeliumSamplesInAssets}.meta";
+        private static readonly Version HeliumUnityAdsSupportedVersion = new Version(4, 2, 1);
+        private static readonly string UnityAdsSDKCommented = $"<!-- <androidPackage spec=\"com.unity3d.ads:unity-ads:{HeliumUnityAdsSupportedVersion}\"/> -->";
+        private static readonly string UnityAdsSDKUncommented = $"        <androidPackage spec=\"com.unity3d.ads:unity-ads:{HeliumUnityAdsSupportedVersion}\"/>";
+
+        /// <summary>
+        /// Finds a package in the Unity project non-restricted to the Unity Registry. Any package on the package.json file can be loaded with this method.
+        /// </summary>
+        /// <param name="packageName">Name of the package to fetch</param>
+        /// <returns>PackageInfo of the found package, null if not found</returns>
+        public static PackageInfo FindPackage(string packageName)
+        {
+            var packageJsons = AssetDatabase.FindAssets("package")
+                .Select(AssetDatabase.GUIDToAssetPath).Where(x => AssetDatabase.LoadAssetAtPath<TextAsset>(x) != null)
+                .Select(PackageInfo.FindForAssetPath).ToList();
+
+            return packageJsons.Find(x => x.name == packageName);
+        }
+
+        /// <summary>
+        /// Imports a sample in the Helium Unity SDK package
+        /// </summary>
+        /// <param name="sampleName">Sample to include into project</param>
+        /// <param name="version">Helium package version to use, must coincide with the currently installed version.</param>
+        /// <returns>Import success status</returns>
+        public static bool ImportSample(string sampleName, string version)
+        {
+            var sample = Sample.FindByPackage(HeliumPackageName, version).Single(x => x.displayName.Equals(sampleName));
+            return sample.Import(ImportOptions.HideImportWindow | ImportOptions.OverridePreviousImports);
+        }
+
+        /// <summary>
+        /// Re-imports a series of Samples based of a collection of Samples names. This is to only update what it's currently in place regardless of the version.
+        /// </summary>
+        /// <param name="existingSamples">Existing Samples to re-import regardless of the version. Name based</param>
+        /// <param name="version">Helium package version to use, must coincide with the currently installed version.</param>
+        public static void ReimportExistingHeliumSamples(ICollection<string> existingSamples, string version)
+        {
+            Directory.Delete(HeliumSamplesInAssets, true);
```

Comment:
If nothing has been created and an re-import is initiated, will this cause any issues?

Reply:
Oh yes, I am addressing that now 👍🏻